backend/src/routes/authRouter.ts

```diff
@@ -1,12 +1,8 @@
 import express from 'express'
 const router = express.Router()
 
-import { registerUser, loginUser, updateUser } from '../controllers/authController'
-import {CatController} from '../cats'
+import { registerUser } from '../controllers/auth'
 
 router.route('/register').post(registerUser)
-router.route('/login').post(loginUser)
```

Comment:
add separation of routes, it can be separate file or just separate line

// User Routes
router.route('/register').post(registerUser)

// Auth Routes
router.route('/login').post(loginUser)**

// Other specific routes
...